slidechain/cmd/peg/peg.go

```diff
@@ -21,12 +21,13 @@ import (
 
 	"github.com/interstellar/slingshot/slidechain"
 	"github.com/interstellar/slingshot/slidechain/stellar"
+	"github.com/interstellar/starlight/worizon/xlm"
 )
 
 func main() {
 	var (
 		custodian   = flag.String("custodian", "", "Stellar account ID of custodian account")
-		amount      = flag.String("amount", "", "amount to peg, in lumens")
+		amount      = flag.Int64("amount", -1, "amount to peg, in lumens")
```

Comment:
Can this not be fractional? And if it can't, should it be in stroops?

IMO the best option is that this remains a string, but one parseable by `xlm.Parse` (from `worizon`).